src/ts-node.ts

```diff
@@ -130,7 +130,7 @@ const DEFAULT_OPTIONS: Options = {
  */
 export function register (opts?: Options) {
   const cwd = process.cwd()
-  const options = extend(DEFAULT_OPTIONS, { project: cwd }, opts)
+  const options = extend(DEFAULT_OPTIONS, opts)
```

Comment:
@blakeembrey I don't think re-ordering will work. The testing that I did showed "extend" would overlay an "undefined" if provided. So if DEFAULT_OPTIONS was overlayed on top of {project:cwd} and no environment variable was set, then the options would end up with undefined in project. I could move it down into this method and say "if (!options.project) { options.project = cwd; }" but that seemed less straightforward than putting it in the initialization of DEFAULT_OPTIONS. Thoughts?